components/yaml/introduction.rst

```diff
@@ -219,6 +222,55 @@ representation to the inline one:
         foo: bar
         bar: baz
 
+Indentation
+"""""""""""
+
+By default the YAML component will use 4 spaces for indentation, this can be
+changed using the second argument as follows:
+
+.. code-block:: php
```

Comment:
please use the `::` shortcut here (meaning the colon on line 229 should be changed to a double one and line 230 and 231 should be removed)